### PR TITLE
Dop 1448 save n get time restriction settings

### PR DESCRIPTION
**Pull request by leoslopez**
Modify endpoints behavior to get and save the new time restriction settings into the contact policies settings.

Already available in INT: https://apisint.fromdoppler.net/contact-policies/swagger/index.html

Associated ticket: [DOP-1448](https://makingsense.atlassian.net/browse/DOP-1448)

[DOP-1448]: https://makingsense.atlassian.net/browse/DOP-1448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by RodrigoPereyraDiaz — CHANGES_REQUESTED** [review]

File: Doppler.ContactPolicies.Business.Logic.Test/ContactPoliciesServiceTest.cs [hunk 1]
```diff
@@ -36,5 +43,81 @@ public async Task
...
+                UserHasContactPolicies = false,
+                ExcludedSubscribersLists = excludedLists,
+            };
+            var contactPoliciesTimeRestrictionExpected = new ContactPoliciesTimeRestriction()
+            {
+                TimeSlotEnabled = true,
+                HourFrom = 0,
+                HourTo = 23,
+                WeekdaysEnabled = false,
+            };
```
Comment (RodrigoPereyraDiaz) [inline Doppler.ContactPolicies.Business.Logic.Test/ContactPoliciesServiceTest.cs:90]: This could be created with _Fixture_

**Review by RodrigoPereyraDiaz — APPROVED** [review]
Good.
Try add more tests to keep the current coverage or increment it

File: Doppler.ContactPolicies.Business.Logic.Test/ContactPoliciesServiceTest.cs (hunk 1 above)
Comment (leoslopez) [reply Doppler.ContactPolicies.Business.Logic.Test/ContactPoliciesServiceTest.cs]: Done: https://github.com/FromDoppler/doppler-contact-policies-api/pull/180/commits/be7027cd25bf622d8c3e04f8b67603f0c48bd7c6

**Comment by leoslopez** [comment]
> Good.
> Try add more tests to keep the current coverage or increment it

We have talked with @RodrigoPereyraDiaz and decided to add testing for repositories as part of another task, because we would need to encapsulate db behavior (connections and queries) in another class.

**Pull request merged**